app/web/src/store/asset.store.ts

```diff
@@ -91,23 +90,22 @@ export const useAssetStore = () => {
       state: () => ({
         assetList: [] as AssetListEntry[],
         assetsById: {} as Record<AssetId, Asset>,
-        selectedAssetId: null as AssetId | null,
         openAssetFuncIds: {} as { [key: AssetId]: FuncId[] },
       }),
       getters: {
         assets: (state) => _.values(state.assetsById),
         selectedAsset(): Asset | undefined {
           return this.assetsById[this.selectedAssetId || 0];
         },
-        urlSelectedAssetId(): AssetId | undefined {
+        selectedAssetId(): AssetId | undefined {
```

Comment:
In the other stores, I've been careful to leave the name as `urlSelectedThingId` to make it clear that this is the one selected via the URL only. Then I usually have another `selectedThingId` which is the id of the thing once we have verified the id is good and loaded the thing.

This helps deal with the situation where there is an ID in the url but it isn't good (either totally bogus, or thing has been deleted, no access, etc...)